<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cliente</class>
 <widget class="QDialog" name="cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>* CPF:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>E-mail:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>* Nome:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCPF">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="inputMask">
    <string>999.999.999-99</string>
   </property>
   <property name="text">
    <string>..-</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Cliente</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEmail">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNome">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Os campos marcados com (*) são obrigatórios.</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtNome</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>txtCPF</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
